test/support/parser.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Membrane.WebM.Debug.Parser do
```

Comment:
Mark those helper modules as to be removed as you probably just need for personal debuging.